hack/config

```diff
@@ -5,7 +5,7 @@ CDI_VERSION="v1.17.0"
 NETWORK_ADDONS_VERSION="0.36.0"
 SSP_VERSION="v1.0.30"
 NMO_VERSION="v0.5.0"
-HPPO_VERSION="v0.2.8"
+HPPO_VERSION="v0.4.2"
 HPP_VERSION="v0.2.2"
```

Comment:
We probably have also to update this one to: v0.4.0

Reply:
this would be in a separate PR, in next run